pages/linux/updatedb.md

```diff
@@ -0,0 +1,12 @@
+# updatedb
+
+> Create or update the database used by `locate`.
+> It usually runs daily by cron.
```

Comment:
Perhaps `> It is usually run daily by cron.` would read better?

Reply:
Hmm, I'm not sure. English is my second language and personally I think that the current version is simpler to understand (active x passive voice). That being said, I'm ok with changing to the suggested version if that's what everybody agrees upon.

Reply:
Ah, I see. The current structure of the sentence indicates that the command is running past cron, which doesn't make sense in this instance. Rather, we want to indicate that cron is the one running the command. To do this, we can alter the structure of the sentence as above.